src/main/java/com/conveyal/datatools/manager/jobs/feedmerge/StopsMergeLineContext.java

```diff
@@ -33,6 +34,25 @@ public boolean checkFieldsForMergeConflicts(Set<NewGTFSError> idErrors, FieldCon
         return checkRoutesAndStopsIds(idErrors, fieldContext);
     }
 
+    @Override
+    public void checkFieldsForReferences(FieldContext fieldContext) {
+        updateParentStationReference(fieldContext);
+    }
+
+    /**
+     * If there is a parent station reference, update to include the scope stop_id.
+     */
+    private void updateParentStationReference(FieldContext fieldContext) {
+        if (fieldContext.nameEquals("parent_station")) {
+            String parentStation = fieldContext.getValue();
+            if (!"".equals(parentStation)) {
+                LOG.debug("Updating parent station to: " + getIdWithScope(parentStation));
```

Comment:
```suggestion
                LOG.debug("Updating parent station to: {}", getIdWithScope(parentStation));
```

With a single parameter it doesn't make much of a difference, but I like this style better.